docs/AzureDevOps/DefinitionReference.md

```diff
@@ -289,11 +291,11 @@ public override SingleStagePipeline Pipeline => new()
 
 See more examples in the [test class](../../tests/Sharpliner.Tests/AzureDevOps/TemplateTests.cs#49)
 
-## Conditioned expressions
+## Conditional expressions
```

Comment:
this is referring to the [`Conditioned`](https://github.com/sharpliner/sharpliner/blob/main/src/Sharpliner/AzureDevOps/ConditionedExpressions/Conditioned.cs) type

Reply:
Yeah, I did wonder about that.

When I read the docs from the perspective of a new user who is familiar with AzDO, then using the AzDO terminology made sense to me.

But if you want to stick with 'Conditioned' I can revert that change.

Reply:
Let's wait for @premun

Reply:
Is the AzDO terminology using "conditional"?

Reply:
Yes. See https://learn.microsoft.com/azure/devops/pipelines/process/template-expressions?view=azure-devops&WT.mc_id=DOP-MVP-5001655#conditional-insertion